rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -760,51 +782,53 @@ void main_window::InstallPup(QString file_path)
 			QMessageBox::Yes | QMessageBox::No, QMessageBox::No) != QMessageBox::Yes)
 		{
 			gui_log.notice("Firmware: Cancelled installation from drop. File: %s", sstr(file_path));
-			return;
+			return false;
 		}
 	}
 
-	if (!file_path.isEmpty())
-	{
-		// Handle the actual installation with a timeout. Otherwise the source explorer instance is not usable during the following file processing.
-		QTimer::singleShot(0, [this, file_path]()
```

Comment:
this is not to be changed. we don't want to block the file explorer.

Reply:
Removed.